src/SixLabors.Fonts/TextLayout.cs

```diff
@@ -161,7 +162,18 @@ public IReadOnlyList<GlyphLayout> GenerateLayout(ReadOnlySpan<char> text, Render
                         location.X = glyphLocation.X;
                     }
 
-                    layout.Add(new GlyphLayout(codePoint, new Glyph(glyph, spanStyle.PointSize), glyphLocation, glyphWidth, glyphHeight, lineHeight, startOfLine, false, false));
+                    foreach (var g in glyphs)
+                    {
+                        var w = (g.AdvanceWidth * spanStyle.PointSize) / scale;
```

Comment:
Is the Fonts library using the same standards as others? I can't tell what type w without loading up the IDE.

Reply:
should be... its using the 'shared-infrastructure' repo.

It does however have a subtly different .editorconfig so that could be the cause of the differences.

Reply:
Yeah that makes sense. There's some known bugginess in the way editorconfig determines when `var` should be used with built in types. The config from the shared infrastructure contains the workaround. I've set ImageSharp to automatically copy taht file and the gitattributes file to the root.